src/AppBundle/Resources/views/Report/Document/deleteConfirm.html.twig

```diff
@@ -0,0 +1,40 @@
+{% extends 'AppBundle:Layouts:application.html.twig' %}
+
+{% set translationDomain = "report-documents" %}
+{% trans_default_domain translationDomain %}
+{% set page = "removeDocument" %}
+
+{% block htmlTitle %}{{ (page ~ '.htmlTitle') | trans }}{% endblock %}
+{% block pageTitle %}{{ (page ~ '.h1Title') | trans }}{% endblock %}
+{% block pageTitleClass %}heading-large{% endblock %}
+
+{% block supportTitleTop %}
+    <span class="heading-secondary">{{ (page ~ '.supportTitle') | trans }}</span>
+{% endblock %}
+
+{% block breadcrumbs %}
+    {{ macros.breadcrumbs(report) }}
```

Comment:
does this need breadcrumbPa ? I'm not up to date with those breadcbrumbs but I saw a PA one around once